app/services/disability_claim_service.rb

```diff
@@ -16,7 +16,7 @@ def all
       next unless raw_claims[key]
       claims << raw_claims[key].map do |raw_claim|
         create_or_update_claim(raw_claim)
-      end
+      end.compact
```

Comment:
I think it'd be cleaner to filter out the claim before mapping to a claim object rather than altering the return type of the `create_or_update_claim` method, something like
```ruby
raw_claims[key].select do |raw_claim|
  DISABILITY_BENEFIT_CODES.include? raw_claim['benefitClaimTypeCode']
end.map do |raw_claim|
...
```